robot/control/Src/modules/LEDModule.cpp

```diff
@@ -168,4 +195,44 @@ void LEDModule::setColor(uint32_t led0, uint32_t led1) {
     dotStarNCS.write(false);
     dotStarSPI.lock()->transmit(data);
     dotStarNCS.write(true);
-}
\ No newline at end of file
+}
+
+void LEDModule::displayErrors() {
+    // If there are no errors, then there is no point in displaying anything
+    if (std::find(errToggles.begin(), errToggles.end(), true) == errToggles.end()) {
```

Comment:
Could you use [none_of](https://en.cppreference.com/w/cpp/algorithm/all_any_none_of) here?

Reply:
Yes, that's a nice expression!